app/assets/stylesheets/forever_style_guide/modules/_hero.scss

```diff
@@ -70,7 +74,7 @@ TODO: use picture element + picturefill to avoid css hacks
   background-repeat: no-repeat;
 
   @media (min-width: $screen-xs) {
-    padding-top: 34%; /* (484 / 1400 * 100) image sizes: 768px x 266px (xs) and 1400px x 484px (sm) */
+    padding-top: percentage($hero-img-height / $hero-img-width); /* image sizes: 768px x 266px (xs) and 1400px x 484px (sm) */
```

Comment:
Kinda, but it doesn't feel explicit. If someone's starting from the class name they see in the markup, they have to scroll up and check the actual dimensions and it feels to me like piecing something together as opposed to just glancing and knowing immediately oh, these are the images we need? It's also not clear from this code that we're saving out two versions of the second image to help keep file sizes down. On the other hand, I've already told Megan what our guidelines are, so I can see an argument that we don't need to worry about it much.